dataplex/snippets/src/test/java/dataplex/EntryGroupIT.java

```diff
@@ -0,0 +1,118 @@
+    return value;
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireProjectIdEnvVar();
+  }
+
+  @BeforeClass
+  // Set-up code that will be executed before all tests
+  public static void setUp() throws Exception {
+    String entryGroupId = "test-entry-group" + ID;
+    locationName = LocationName.of(PROJECT_ID, LOCATION);
+    entryGroupName = EntryGroupName.of(PROJECT_ID, LOCATION, entryGroupId);
+    expectedEntryGroup =
+        String.format(
+            "projects/%s/locations/%s/entryGroups/%s", PROJECT_ID, LOCATION, entryGroupId);
+    // Create Entry Group resource that will be used in tests for "get", "list" and "update" methods
+    CreateEntryGroup.createEntryGroup(locationName, entryGroupId);
+  }
+
+  @Test
+  public void listEntryGroups_returnsListContainingEntryGroupCreatedInSetUp() throws IOException {
+    List<EntryGroup> entryGroups = ListEntryGroups.listEntryGroups(locationName);
+    assertThat(entryGroups.stream().map(EntryGroup::getName)).contains(expectedEntryGroup);
+  }
+
+  @Test
+  public void getEntryGroup_returnsEntryGroupCreatedInSetUp() throws IOException {
+    EntryGroup entryGroup = GetEntryGroup.getEntryGroup(entryGroupName);
+    assertThat(entryGroup.getName()).isEqualTo(expectedEntryGroup);
+  }
+
+  @Test
+  public void updateEntryGroup_returnsUpdatedEntryGroup() throws Exception {
+    EntryGroup entryGroup = UpdateEntryGroup.updateEntryGroup(entryGroupName);
+    assertThat(entryGroup.getName()).isEqualTo(expectedEntryGroup);
+  }
+
+  @Test
+  public void createEntryGroup_returnsCreatedEntryGroup() throws Exception {
+    String entryGroupIdToCreate = "test-entry-group" + UUID.randomUUID().toString().substring(0, 8);
+    EntryGroupName entryGroupNameToCreate =
+        EntryGroupName.of(PROJECT_ID, LOCATION, entryGroupIdToCreate);
+    String expectedEntryGroupToCreate =
+        String.format(
+            "projects/%s/locations/%s/entryGroups/%s", PROJECT_ID, LOCATION, entryGroupIdToCreate);
+
+    EntryGroup entryGroup = CreateEntryGroup.createEntryGroup(locationName, entryGroupIdToCreate);
+    // Clean-up created Entry Group
+    DeleteEntryGroup.deleteEntryGroup(entryGroupNameToCreate);
+
+    assertThat(entryGroup.getName()).isEqualTo(expectedEntryGroupToCreate);
+  }
+
+  @Test
+  public void deleteEntryGroup_executesTheCallWithoutException() throws Exception {
+    String entryGroupIdToDelete = "test-entry-group" + UUID.randomUUID().toString().substring(0, 8);
+    EntryGroupName entryGroupNameToDelete =
+        EntryGroupName.of(PROJECT_ID, LOCATION, entryGroupIdToDelete);
+    // Create Entry Group to be deleted
+    CreateEntryGroup.createEntryGroup(locationName, entryGroupIdToDelete);
+
+    // No exception means successful call
+    DeleteEntryGroup.deleteEntryGroup(entryGroupNameToDelete);
+  }
+
+  @AfterClass
+  // Clean-up code that will be executed after all tests
+  public static void tearDown() throws Exception {
```

Comment:
method names use snake notation which is not used in Java. Please, use pascal notation.